pkg/flow/reconciler/xslttransform/reconciler.go

```diff
@@ -35,17 +37,26 @@ type reconciler struct {
 
 	// Knative Service reconciler
 	ksvcr libreconciler.KServiceReconciler
+
+	sinkResolver *resolver.URIResolver
 }
 
 // Check that our Reconciler implements Interface
 var _ reconcilerv1alpha1.Interface = (*reconciler)(nil)
 
 // ReconcileKind implements Interface.ReconcileKind.
 func (r *reconciler) ReconcileKind(ctx context.Context, o *v1alpha1.XSLTTransform) pkgreconciler.Event {
-	o.Status.InitializeConditions()
-	o.Status.ObservedGeneration = o.Generation
+	var url *apis.URL
+
+	if o.Spec.Sink != nil {
+		uri, err := r.resolveDestination(ctx, o)
```

Comment:
nit: We could as well assign directly to `url` instead of having this strange `url = uri` below.

Reply:
This wasn't resolved.
(Or rather, resolved in a different component unrelated to this PR)

Reply:
got it now!